README.md

```diff
@@ -1,2 +0,0 @@
-# segmentation
-Implémentation de plusieurs algorithmes de segmentation d'images
```

Comment:
sorry, but you should not change this file. It describes the whole segmentation project (and you're not alone on this project). Please, revert your changes.